libs/designsystem/src/lib/components/form-field/directives/numeric/numeric-input.directive.spec.ts

```diff
@@ -0,0 +1,321 @@
+import { CurrencyPipe, DecimalPipe } from '@angular/common';
+import { LOCALE_ID } from '@angular/core';
+import { FormControl, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { createDirectiveFactory, SpectatorDirective } from '@ngneat/spectator';
+
+import { NumericInputDirective } from './numeric-input.directive';
+
+describe('Directive: NumericInputDirective', () => {
+  [
+    //    { id: 'da', separators: { group: '.', decimal: ',' } },
```

Comment:
Should be either active or removed, and since it's a foreach it should properly be active

Reply:
Good poijt. I forgot to handle it. The da locale is not behaving properly

Reply:
I have (again) been trying to register da as a locale, just for the testing purposes. No success.